src/renderers/WebGLRenderer.js

```diff
@@ -120,7 +120,22 @@ function WebGLRenderer( parameters = {} ) {
 
 	// physically based shading
 
-	this.outputEncoding = LinearEncoding;
+	let _outputEncoding = LinearEncoding;
+
+	Object.defineProperty( this, 'outputEncoding', {
+
+		get: () => _outputEncoding,
+
+		set: ( encoding ) => {
+
+			_outputEncoding = encoding;
+
+			// Reapply color space conversion.
+			background.setClearColor( background.getClearColor() );
```

Comment:
This line feels odd. Probably the background color should be resolved to the correct color space for a given render target, instead of here.